src/common/storage/src/location.rs

```diff
@@ -156,8 +156,9 @@ pub fn parse_uri_location(l: &UriLocation) -> Result<(StorageParams, String)> {
                     .unwrap_or_default(),
                 security_token: l
```

Comment:
how about make the variable name into session_token together?

Reply:
We can't. It has been persisted in meta.

Reply:
the internal remains the same, otherwise, protobuf stuff should be dealt with.